src/uct/tcp/tcp_iface.c

```diff
@@ -46,14 +46,19 @@ static ucs_config_field_t uct_tcp_iface_config_table[] = {
    ucs_offsetof(uct_tcp_iface_config_t, prefer_default), UCS_CONFIG_TYPE_BOOL},
 
   {"CONN_NB", "n",
-   "Enables non-blocking connection establishment. It may improve startup "
+   "Enable non-blocking connection establishment. It may improve startup "
    "time, but can lead to connection resets due to high load on TCP/IP stack",
    ucs_offsetof(uct_tcp_iface_config_t, conn_nb), UCS_CONFIG_TYPE_BOOL},
 
   {"MAX_POLL", UCS_PP_MAKE_STRING(UCT_TCP_MAX_EVENTS),
    "Number of times to poll on a ready socket. 0 - no polling, -1 - until drained",
    ucs_offsetof(uct_tcp_iface_config_t, max_poll), UCS_CONFIG_TYPE_UINT},
 
+  {UCT_TCP_CONFIG_MAX_CONN_ATTEMPTS, "5",
```

Comment:
consider using the word "retries" instead of "attempts" all over, i think it's more standard terminology for networking

Reply:
done